terraform/vars.tf

```diff
@@ -12,28 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# The default region for resources in the project, individual resources should
+# have more specific variables defined to specify their regions to increase the
+# flexibility of deployments
 variable "region" {
   type    = string
   default = "us-central1"
 }
 
+# The region in which to put the SQL DB - it is currently configured to use
+# PostgreSQL:
+# https://cloud.google.com/sql/docs/postgres/locations
+variable "db_region" {
+  type = string
+  default = "us-central1"
+}
+
+# The region in which to put the key management service:
+# https://cloud.google.com/kms/docs/locations
+variable "kms_location" {
+  type = string
+  default = "us-central1"
+}
+
+# The location for the app engine; this includes scheduler jobs for the app
+# engine as defined by the appengine_region local, derived from this variable:
+# https://cloud.google.com/appengine/docs/locations
 variable "appengine_location" {
```

Comment:
We have appengine_region and appengine_location now. Can we reduce this to one?

Reply:
Not without some hidden string manipulation - I used appengine_region for scheduler_jobs. If appengine_location  == "us-central" then appengine_region must equal "us-central1". The same is true for "eu-west" and "eu-west1". Otherwise they can be the same (at the moment).

Reply:
Hmm okay. It seems weird to have two "appengine" variables. Can we name the other one "cloudscheduler_location" instead of appengine_region and document that it has to co-locate with appengine?

Reply:
OK, I'll have to make that change my tomorrow morning.

Reply:
I changed it to cloudscheduler_region to match the "region" property of the resource. Ready for review.